src/Aquifer.Public.API/Endpoints/Bibles/List/Endpoint.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, IReadOnlyList<Response>>
+{
+    private readonly AquiferDbContext _dbContext = dbContext;
+
+    public override void Configure()
+    {
+        Get("/bibles");
+        Description(d => d.ProducesProblemFE());
+        Summary(s =>
+        {
+            s.Summary = "Get a list of Bibles.";
+            s.Description =
+                "For a given optional language ID, returns the Bibles in the system including language information, abbreviation codes, whether or not the Bible supports a Greek alignment, and license information.";
```

Comment:
I know this is a massive nitpick, but we're just saying "id" in most places I think, rather than "ID". Would prefer to be consistent with our grammar regardless of what we go with.

Reply:
It's not a massive nitpick.  At my last job I made everyone use `FooId`/`fooId` for code naming but use `Foo ID` for human readable strings and was notorious for leaving that in code review. 😄 I can argue at length for why [ID](https://en.wikipedia.org/wiki/ID) is correct over "Id", "Id.", and "id" but I'll spare you. 🤣 

I'll switch to `id` instead for consistency.  I just might accidentally slip in an `ID` once in a while because old habits die hard.

Reply:
Fix [commit](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/15bbd09583a8162999f91dca61b65816c7142a23).